eclair-core/src/main/resources/reference.conf

```diff
@@ -82,7 +82,8 @@ eclair {
 
     reserve-to-funding-ratio = 0.01 // recommended by BOLT #2
     max-reserve-to-funding-ratio = 0.05 // channel reserve can't be more than 5% of the funding amount (recommended: 1%)
-    min-funding-satoshis = 100000
+    min-public-funding-satoshis = 100000 // high public minimums are attractive to pathfinding heuristics
+    min-private-funding-satoshis = 100000 // safe private minimums increase the cost of spamming and flood-and-loot attacks
```

Comment:
I don't think these comments are really helpful. Larger channels are more attractive but that doesn't have anything to do with the minimum we set here, a small channel is more attractive than no channel.

Reply:
Removed them in [6534977](https://github.com/ACINQ/eclair/pull/2203/commits/6534977f9f6d81813c31190663da069d230c0343). Sounds like its too nuanced to explain in comments.